Source/Charts/Renderers/PieChartRenderer.swift

```diff
@@ -119,10 +119,14 @@ open class PieChartRenderer: DataRenderer
         let phaseX = animator.phaseX
         let phaseY = animator.phaseY
 
+        let borderWidth = dataSet.sliceBorderWidth
+        let borderColor = dataSet.sliceBorderColor
+        let drawBorder = borderWidth > 0.0
+        
         let entryCount = dataSet.entryCount
         let drawAngles = chart.drawAngles
         let center = chart.centerCircleBox
-        let radius = chart.radius
+        let radius = chart.radius - borderWidth
```

Comment:
why reduce borderWidth here? I remember this is unnecessary? The border is rendered like
| half border width | arc(at the radius length) | half border width |

Or we want the boder width is included with the radius? However if this is the case, the real radius should be 
>chart.radius - borderWidth/2?

@jjatie correct me if I'm wrong

Reply:
I think you're right @liuxuan30

Reply:
Which one? With or without `-borderWidth/2 `?

Reply:
Without

Reply:
I have thought it again, that people could? use the the bounding rect's min(width, height) serving as radius. In this extreme case, if the border is enabled, we will see the outside part of the border is cut off.

So I guess we still need to use `chart.radius - borderWidth/2` as the real radius?